src/test/java/com/learning/selenium/pages/GoogleCloudPricingCalcPage.java

```diff
@@ -0,0 +1,97 @@
+
+    @FindBy(css = "div[title='Compute Engine']")
+    private WebElement computeEngine;
+    private final String inputFieldBaseLocator = "//label[contains(.,'%s')]/../input";
+    private final String selectFieldBaseLocator = "//md-select[@ng-model='%s']";
+    private final String selectOptionBaseLocator = ".//md-option//div[contains(., '%s')]";
+    private final String selectOptionBaseLocatorSpecific = "//md-option[@*[contains(.,'%s')]]//div[contains(., '%s')]";
+
+    private final By numberOfInstancesField = LocatorHelper.getLocatorFromOneValue(inputFieldBaseLocator, "Number of instances");
+    private final By whatAreInstancesField = LocatorHelper.getLocatorFromOneValue(inputFieldBaseLocator, "What are these instances for?");
+    private final String operatingSystemUnique = "listingCtrl.computeServer.os";
+    private final By operatingSystem =  LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, operatingSystemUnique);
+    private final String provisioningModelUnique = "listingCtrl.computeServer.class";
+    private final By provisioningModel = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, provisioningModelUnique);
+    private final String seriesUnique = "listingCtrl.computeServer.series";
+    private final By series = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, seriesUnique);
+    private final String machineTypeUnique = "listingCtrl.computeServer.instance";
+    private final By machineType = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, machineTypeUnique);
+
+    @FindBy(xpath = "//*[contains(@aria-label, 'Add GPUs')]")
+    private WebElement addGpus;
+    private final String gpuTypeUnique = "listingCtrl.computeServer.gpuType";
+    private final By gpuType = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, gpuTypeUnique);
+    private final String numberOfGpusUnique = "listingCtrl.computeServer.gpuCount";
+    private final By numberOfGpus = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, numberOfGpusUnique);
+    private final String localSsdUnique = "listingCtrl.computeServer.ssd";
+    private final By localSsd = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, localSsdUnique);
+    private final String dataCenterLocationUnique = "listingCtrl.computeServer.location";
+    private final String dataCenterLocationSelect = "computeServer";
+    private final By dataCenterLocation = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, dataCenterLocationUnique);
+    private final String commitedUsageUnique = "listingCtrl.computeServer.cud";
+    private final By commitedUsage = LocatorHelper.getLocatorFromOneValue(selectFieldBaseLocator, commitedUsageUnique);
+
+    @FindBy(xpath = "//*[contains(@aria-label, 'Add to Estimate')]")
+    private WebElement addToEstimateButton;
+    private By estimateSection = By.cssSelector(".cartitem");
+    private By estimateSectionRows = By.cssSelector(".md-1-line");
```

Comment:
according to the code convention fields should go before constructor or methods

Reply:
Updated in Task2